### ISSUE TITLE
Recent firmware causes pan camera to stop being found

### ISSUE THREAD

**Issue by kevin182** [mrlt8/docker-wyze-bridge#31]
My firmware recently got updated for my v3 and pan camera. After updating to the latest container, it still will not find the pan camera.

**Comment by kevin182** [comment]
2021/07/27 14:30:45 I [1/1] [RTSP] [conn xxx.xxx.xxx.xxx:yyyyy] ERR: no one is publishing to path 'garage-cam'
2021/07/27 14:30:45 I [1/1] [RTSP] [conn xxx.xxx.xxx.xxx:yyyyy] closed

2021/07/27 14:30:48 [Garage Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
2021/07/27 14:30:48 [Garage Cam] Camera offline? Sleeping for 10s.

This just repeats

**Comment by mrlt8** [comment]
Does the v3 on the new firmware still work?

Could you try adding this to your docket-compose.yml to see if it helps:

```Yaml
- FRESH_DATA=True
```

**Comment by kevin182** [comment]
```python
version: '3.8'
services:
    wyze-bridge:
        restart: always
        # network_mode: host
        ports:
            - 1935:1935
            - 8554:8554
            - 8888:8888
        build: 
            context: ./app
            # dockerfile: Dockerfile.arm
        environment:
            - WYZE_EMAIL=xxxx@zzz
            - WYZE_PASSWORD=yyyy
            - FRESH_DATA=True
```
This is still not working.

**Comment by mrlt8** [comment]
It seems like the pan cameras can be hit or miss for some reason.
Were you able to stream from it before?

**Comment by kevin182** [comment]
It was working before.

**Comment by kevin182** [comment]
This still does not work. I do not have the rtsp firmware on my pan cam either.

**Comment by mrlt8** [comment]
You'll need to post more info as `IOTC_ER_CAN_NOT_FIND_DEVICE` usually means the camera is offline even in the app.

**Comment by kevin182** [comment]
Please let me know what you need and I will get it to you. Thank you.

**Comment by jims777** [comment]
I'm getting this too since updating a v2 and pan cam to the newest non-RTSP firmware, 4.9.7.798 and 4.10.7.798, respectively. After the firmware update, I think Tinycam gave the same error on these cameras too until I restarted the app.

> 
wyze-bridge_1  | 2021/08/01 21:21:54 [Garage Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
wyze-bridge_1  | 2021/08/01 21:21:54 [Garage Cam] Camera offline? Will retry again in 10s.
wyze-bridge_1  | 2021/08/01 21:21:54 [Porch Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
wyze-bridge_1  | 2021/08/01 21:21:54 [Porch Cam] Camera offline? Will retry again in 10s.
wyze-bridge_1  | 2021/08/01 21:22:19 [Garage Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
wyze-bridge_1  | 2021/08/01 21:22:19 [Garage Cam] Camera offline? Will retry again in 20s.
wyze-bridge_1  | 2021/08/01 21:22:19 [Porch Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
wyze-bridge_1  | 2021/08/01 21:22:19 [Porch Cam] Camera offline? Will retry again in 20s.
>

**Comment by mrlt8** [comment]
Could you see if v0.5.0 helps?

**Comment by jims777** [comment]
It's giving "no one is publishing" and "path must end with slash" errors on the v2 and pan now. The "can not find device" error has gone away.

wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [conn 192.168.0.109:55927] ERR: no one is publishing to path 'porch-cam'
wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [conn 192.168.0.109:55927] closed
wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [conn 192.168.0.109:55928] opened
wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [session 763143545] opened by 192.168.0.109:55928
wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [session 763143545] closed
wyze-bridge_1  | 2021/08/03 07:04:26 I [4/0] [RTSP] [conn 192.168.0.109:55928] ERR: path must end with a slash (porch-cam)
...
wyze-bridge_1  | 2021/08/03 07:04:42 I [4/0] [RTSP] [conn 192.168.0.109:55946] ERR: no one is publishing to path 'garage-cam'
wyze-bridge_1  | 2021/08/03 07:04:42 I [4/0] [RTSP] [conn 192.168.0.109:55946] closed
wyze-bridge_1  | 2021/08/03 07:04:42 I [4/0] [RTSP] [conn 192.168.0.109:55947] opened
wyze-bridge_1  | 2021/08/03 07:04:42 I [4/0] [RTSP] [session 3236600401] opened by 192.168.0.109:55947
wyze-bridge_1  | 2021/08/03 07:04:42 I [4/0] [RTSP] [conn 192.168.0.109:55947] ERR: path must end with a slash (garage-cam)

**Comment by mrlt8** [comment]
Do you see the problem cameras get started by the script? It should say something like `is publishing to path 'garage', 1 track with UDP`

"no one is publishing" just means you're trying to consume the stream before it gets published.

**Comment by jims777** [comment]
No, the v2 and pan with the new firmware never start up, only my v3's. They did prior to the firmware upgrade.

wyze-bridge_1  | 2021/08/03 17:41:18 [Northeast Cam] Starting HD 120kb/s Stream for WyzeCam V3 (WYZE_CAKP2JFUS) in "LAN mode" FW: 4.36.2.5 IP: 192.168.0.228 WiFi: 87%
wyze-bridge_1  | 2021/08/03 17:41:18 [Southeast Cam] Starting HD 120kb/s Stream for WyzeCam V3 (WYZE_CAKP2JFUS) in "LAN mode" FW: 4.36.2.5 IP: 192.168.0.238 WiFi: 97%
wyze-bridge_1  | 2021/08/03 17:41:18 [Southwest Cam] Starting HD 120kb/s Stream for WyzeCam V3 (WYZE_CAKP2JFUS) in "LAN mode" FW: 4.36.2.5 IP: 192.168.0.163 WiFi: 97%
wyze-bridge_1  | 2021/08/03 17:41:19 [Back Porch Cam] Starting HD 120kb/s Stream for WyzeCam V3 (WYZE_CAKP2JFUS) in "LAN mode" FW: 4.36.2.5 IP: 192.168.0.162 WiFi: 96%

I didn't wait long enough, the v2 and pan are still reporting as not found:

wyze-bridge_1  | 2021/08/03 17:43:39 [Garage Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
wyze-bridge_1  | 2021/08/03 17:43:39 [Porch Cam] IOTC_ER_CAN_NOT_FIND_DEVICE

**Comment by inactivefriend** [comment]
Having the same issue (IOTC_ER_CAN_NOT_FIND_DEVICE) with Wyze Cam v2 on firmware 4.9.7.798 but once I downgraded to 4.9.6.241 it works flawlessly.

**Comment by jbruening** [comment]
Confirmed, was having the same issue with a v2 camera. Upgraded my camera prior to installing the docker container(mrlt8/wyze-bridge:latest@sha256:0cb21a348cac7afb6ee1fb2c8f9506a47b3d7af55589c2d2a764df82b632eb7a) .  Just downgraded the firmware while the container was running.

```
2021/08/07 00:25:48 [Office Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
2021/08/07 00:26:51 [Office Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
2021/08/07 00:27:54 [Office Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
2021/08/07 00:28:56 [Office Cam] IOTC_ER_CAN_NOT_FIND_DEVICE
2021/08/07 00:29:54 [Office Cam] Starting 1080p 120kb/s Stream for WyzeCam V2 (WYZEC1-JZ) in "LAN mode" FW: 4.9.6.241 IP: 192.168.0.173 WiFi: 81%
2021/08/07 00:29:59 I [0/0] [RTSP] [conn 127.0.0.1:49642] opened
2021/08/07 00:29:59 I [0/0] [RTSP] [session 1788587234] opened by 127.0.0.1:49642
2021/08/07 00:29:59 I [1/0] [RTSP] [session 1788587234] is publishing to path 'office-cam', 1 track with UDP
```

**Comment by Nincodedo** [comment]
I'm also having this issue with firmware 4.9.7.798.

**Comment by edwardcc83** [comment]
Having the same error with both cameras running current firmware (4.9.7.1068)

**Comment by csanner** [comment]
This is a known issue with DTLS in the newest firmware to fix a security issue.

I don't know if there are plans to work around/fix it but in the project README it suggests to not upgrade the firmware.

Too late for me but...*shrugs*
Hopefully we can get past it
